### ISSUE TITLE
Error when try to run migrations

### ISSUE THREAD

**Issue by danilodeveloper** [tgriesser/knex#2855]
# Environment
MacOSX 10.14 (18A391 - Mojave)

Knex version:
_0.15.2_
Database + version:
_PostgreSQL 10_
OS:
_MacOSX 10.14 (18A391 - Mojave)_

# Bug

1. Explain what kind of behaviour you are getting and how you think it should do:
_I just migrate from Knex 0.13.0 to Knex 0.15.12 and my migrations stopped running, if I return to Knex 0.13.0 everything works well.
Even droping all tables I still get the error._

2. Error message:
``` 
knex migrate:latest
Using environment: development
migration failed with error: create table "core"."db_migrations" ("id" serial primary key, "name" varchar(255), "batch" integer, "migration_time" timestamptz) - relation "db_migrations" already exists
error: relation "db_migrations" already exists
    at Connection.parseE (/Users/danilo/Projetos/eductalk/server/node_modules/pg/lib/connection.js:554:11)
    at Connection.parseMessage (/Users/danilo/Projetos/eductalk/server/node_modules/pg/lib/connection.js:379:19)
    at Socket.<anonymous> (/Users/danilo/Projetos/eductalk/server/node_modules/pg/lib/connection.js:119:22)
    at Socket.emit (events.js:182:13)
    at addChunk (_stream_readable.js:283:12)
    at readableAddChunk (_stream_readable.js:264:11)
    at Socket.Readable.push (_stream_readable.js:219:10)
    at TCP.onread (net.js:639:20)
```

**Comment by kibertoad** [comment]
See https://github.com/tgriesser/knex/issues/2499
tl;dr - schemaName for migrations can't be passed as a part of changelogName anymore, pass it as a separate config value `schemaName`

**Comment by danilodeveloper** [comment]
Cool! You solved, thank you @kibertoad !!!

**Comment by kibertoad** [comment]
np, glad I could help!
